### ISSUE TITLE
Request for Python 3.2 support

### ISSUE THREAD

**Issue by pboehm** [PythonCharmers/python-future#29]
Hello,

I've come across this amazing library and found a possible bug on Python 3.2. I found this in a django extension which is tested by Travis against 2.7, 3.2 and 3.3 and on 3.2 the tests fail because of an SyntaxError in future/utils/**init**.py. 

```
Traceback (most recent call last):
...
  File "/home/travis/virtualenv/python3.2/lib/python3.2/site-packages/future/utils/__init__.py", line 419
    return isinstance(obj, type(u''))
                                  ^
SyntaxError: invalid syntax
```

This only occurs on 3.2 as you can see here https://travis-ci.org/pboehm/django-data-migration/builds/16821855 . For me this looks like a weird error because I couldn't see and error in this line ...

**Comment by davidak** [comment]
Python 3.2 is available on Ubuntu 12.04 LTS which is still supported until 2017, but 2.7 is installed by default.

Also PyPy3 is based on Python 3.2.5.

**Comment by jmadler** [comment]
Py3.2 is now EOL, and this request is a bit dated.  Closing! :)
